src/components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -293,7 +271,7 @@ function AuctionDetails(props: Props) {
                             </Card.Body>
                         </Card>
                     </div>
-                    <div className="fixed-bottom">{window.navigator.clipboard ? <div className="btn-bottom"><Button type="primary" onClick={copyClick}>{copyButtonClicked ? copiedIcon : copyIcon}</Button></div> : ""}</div>
+                    <CopyButton copyValue={"/viewauction " + props.auctionUUID} successMessage={<p>Copied ingame link <br /><i>/viewauction {props.auctionUUID}</i></p>} />
```

Comment:
The copy button is now on the lower left wich looks odd.  
Please move it next to the subscribe button.